File: 981-delete-columns-to-make-sorted/delete-columns-to-make-sorted.cpp
class Solution {
public:
    int minDeletionSize(vector<string>& strs) {
        int row = strs.size();
        int col = strs[0].size();
        int count = 0;

        for (int i = 0; i < col; i++) {
            for (int j = 0; j < (row - 1); j++) {
                if (strs[j][i] > strs[j + 1][i]) {
                    count++;
                    break;
                }
            }
        }
        
        return count;
    }
};

File: 981-delete-columns-to-make-sorted/delete-columns-to-make-sorted.py
class Solution:
    def minDeletionSize(self, strs: List[str]) -> int:
        col = len(strs[0])
        row = len(strs)
        count = 0

        for c in range(col):
            for r in range(row - 1):
                if strs[r][c] > strs[r + 1][c]:
                    count += 1
                    break

        return count
